__tests__/components/forecast-summaries.test.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import Enzyme from 'enzyme';
+import ForecastSummaries from '../../src/components/forecast-summaries';
+import ForecastSummary from '../../src/components/forecast-summary';
+
+const forecasts = [{
+  date: 123456,
+  description: 'date1',
+  icon: 'icon1',
+  temperature: {
+    max: 99,
+  },
+},
+
+{
+  date: 65432,
+  description: 'date2',
+  icon: 'icon2',
+  temperature: {
+    max: 888,
+  },
+
+},
+];
+
```

Comment:
If you find yourself rendering the same wrapper more than once you should really use a `beforeEach()`. It would let you add more tests a lot faster.